src/libraries/System.Data.Common/src/System/Data/DataView.cs

```diff
@@ -1195,6 +1248,8 @@ string System.ComponentModel.ITypedList.GetListName(PropertyDescriptor[] listAcc
             return string.Empty;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
```

Comment:
We can mark the interface `ITypedList.GetItemProperties` as RequiresUnreferencedCode. As far as I can tell, these are the only 2 places that we have that implement the interface, and they are both are unsafe. Looking at other implementations (like WinForms), theirs appears unsafe as well.

Reply:
pending in next iteration